ds3-sdk/src/main/java/com/spectralogic/ds3client/metadata/MetadataAccessImpl.java

```diff
@@ -19,76 +19,66 @@
 import com.google.common.collect.ImmutableMap;
 import com.spectralogic.ds3client.helpers.Ds3ClientHelpers;
 import com.spectralogic.ds3client.metadata.interfaces.MetadataStore;
-import com.spectralogic.ds3client.metadata.interfaces.MetadataStoreListener;
-import com.spectralogic.ds3client.utils.MetaDataUtil;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import com.spectralogic.ds3client.utils.Platform;
 
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.attribute.BasicFileAttributes;
 import java.nio.file.attribute.PosixFileAttributes;
 import java.util.Map;
-import java.util.Set;
 
 /**
  * Implementation of MetaDataAcess Interface
  * Used to store meta data on Server
  */
 public class MetadataAccessImpl implements Ds3ClientHelpers.MetadataAccess {
-    static private final Logger LOG = LoggerFactory.getLogger(MetadataAccessImpl.class);
     private final Map<String, Path> fileMapper;
-    private final MetadataStoreListener metadataStoreListener;
 
-    public MetadataAccessImpl(final Map<String, Path> fileMapper , final MetadataStoreListener metadataStoreListener) {
+    public MetadataAccessImpl(final Map<String, Path> fileMapper) {
```

Comment:
Since we are consuming fileMapper without modifying it, let's make it immutable.